build.gradle

```diff
@@ -90,44 +91,44 @@ configurations.all
 
 dependencies
 {
-  compile 'commons-lang:commons-lang:2.6'
-  compile 'commons-io:commons-io:2.5'
-  compile 'net.sf.trove4j:trove4j:3.0.3'
-  compile 'args4j:args4j:2.0.29'
-  compile 'com.apple:AppleJavaExtensions:1.4'
-  compile 'com.miglayout:miglayout:3.7.4'
+  implementation 'commons-lang:commons-lang:2.6'
+  implementation 'commons-io:commons-io:2.5'
+  implementation 'net.sf.trove4j:trove4j:3.0.3'
+  implementation 'args4j:args4j:2.0.29'
+  implementation 'com.apple:AppleJavaExtensions:1.4'
+  implementation 'com.miglayout:miglayout:3.7.4'
```

Comment:
Miglayout should probably be upgraded to the latest version used in the scijava POM

Reply:
Same for the commons stuff 👍

Reply:
I updated all dependencies to the same version as those in the scijiava POM (if present).  Miglayout has been split in 3.  Remarkably, no compiler errors.